android-exoplayer/src/main/java/com/brentvatne/exoplayer/VideoEventEmitter.java

```diff
@@ -242,6 +242,12 @@ void end() {
         receiveEvent(EVENT_END, null);
     }
 
+    void endLiveChannelAsVod() {
```

Comment:
The js devs know about this?

Reply:
This is the result of my discussion with yaroslavlvov. I think he will communicate with JS dev. 
Of course, we can also modify these interface codes during integration testing.